actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -191,6 +191,9 @@
     @Config("orbit.actors.defaultActorTTL")
     private long defaultActorTTL = TimeUnit.MINUTES.toMillis(10);
 
+    @Config("orbit.actors.localAddressCacheTTL")
+    private long localAddressCacheTTL = 0;
```

Comment:
Right, 0 should not be the default.  Since the local address cache expires after last access, I think defaultTTL is a sufficient default.  This would mean that the cache entry would be evicted around the same time of the actor TTL would it not?  Is there a need to keep it around longer?